src/Microsoft.AspNetCore.Razor.Language/DesignTimeOptionsFeature.cs

```diff
@@ -5,20 +5,10 @@
 
 namespace Microsoft.AspNetCore.Razor.Language
 {
-    internal class DesignTimeOptionsFeature : RazorEngineFeatureBase, IRazorParserOptionsFeature, IRazorCodeGenerationOptionsFeature
+    internal class DesignTimeOptionsFeature : RazorEngineFeatureBase, IRazorCodeGenerationOptionsFeature
     {
         public int Order { get; set; }
 
-        public void Configure(RazorParserOptionsBuilder options)
-        {
-            if (options == null)
-            {
-                throw new ArgumentNullException(nameof(options));
-            }
-
-            options.DesignTime = true;
-        }
-
         public void Configure(RazorCodeGenerationOptionsBuilder options)
```

Comment:
Will probably end up moving the code generation options to follow the same pattern as the parser options which would result in this class going away.